### PR TITLE
UI tracking number of pending bundles in client server.

### PR DESCRIPTION
**Pull request by darrensh3n**

### CODE REVIEW

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java [hunk 1]
```diff
@@ -39,6 +40,13 @@ public class ServerUploadFragment extends Fragment {
     private ExecutorService executor = Executors.newFixedThreadPool(2);
     private TransportPaths transportPaths;
 
+
+    private TextView numberBundlestoClient;
+    private TextView numberBundlestoServer;
+    private Button reloadButton;
+
```
Comment (breed) [inline BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java:47]: too many whitespaces (you should use the reformat option in intellij)

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java [hunk 2]
```diff
@@ -39,6 +40,13 @@ public class ServerUploadFragment extends Fragment {
     private ExecutorService executor = Executors.newFixedThreadPool(2);
     private TransportPaths transportPaths;
 
+
```
Comment (breed) [inline BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java:43]: too many spaces

File: BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java [hunk 3]
```diff
@@ -79,6 +97,22 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return mainView;
     }
 
+    private int countFilesInDirectory(File dir) {
+        if (dir.exists() && dir.isDirectory()) {
+            File[] files = dir.listFiles();
+            if (files != null) {
```
Comment (breed) [inline BundleTransport/app/src/main/java/net/discdd/bundletransport/ServerUploadFragment.java:103]: you could simplify this. (in a separate PR) 
var files = dir.listFiles(File::isFile);
return files == null ? 0 : files.length;

File: BundleTransport/app/src/main/res/layout/fragment_server_upload.xml [hunk 4]
```diff
@@ -58,6 +58,68 @@
...
+                    android:gravity="left"
+                    android:layout_marginEnd="2dp"
+                    android:padding="0dp"/>
+
+            <Button
+                    android:id="@+id/reload"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:text="Reload"
```
Comment (breed) [inline BundleTransport/app/src/main/res/layout/fragment_server_upload.xml:114]: you should make this into a string resource. quick fix in the id will do it for you. you should probably make the string "Reload counts"

**Review by pankinkun — APPROVED** [review]

**Pull request merged**